finality-aleph/src/aggregation/mod.rs

```diff
@@ -16,16 +17,37 @@ use crate::{
     Keychain,
 };
 
+/// Either a block hash or a performance report hash. They should never overlap. We assume that
+/// BlockHash and Hash are the same, it has always been this way, but if it ever changes this place
+/// will cause trouble.
+#[derive(PartialEq, Eq, StdHash, Copy, Clone, Debug, Encode, Decode)]
+pub enum EitherHash {
```

Comment:
What about `AggregatedHash`? Or `BroadcastedHash`? More specific name, `EitherHash` sounds like `GenericHash`

Reply:
Changed to `SignableTypedHash`, not sure whether that's better, but accurate.